logic/requirements.py

```diff
@@ -36,7 +36,7 @@ def hasConsumableRequirement(self) -> bool:
 
     def test(self, inventory) -> bool:
         for item in self.__items:
-            if item in inventory:
+            if item == True or item in inventory:
```

Comment:
As this is part of an "inner loop" of the randomizer, this change might have a performance impact. With this many changes, I'm also not clearly seeing why it is needed.

Reply:
think this was mainly to include the use of booleans like with text_clip into the logic file. @kbranch do you know more?

Reply:
Oh, huh, I probably messed up a merge somewhere, that looks like the original implementation of booleans in requirements. I'll try to sort it out and send Dragon a PR.